Governance/tech/drs.md

```diff
@@ -1,6 +1,7 @@
 # Choosing a directory structure (DRS) and filenames
 
-The names you choose for files and directories and generally the way you organise your data, i.e. your directory structure (DRS) can help navigating the data and provide extra information, avoid confusion and the user ending up accessing the wrong data. As for many other cases the best file organisation will depend on the specific research project and the actual server where the data is stored. Here we are just listing a few guidelines and tips to help you decide.  
+The names you choose for files and directories and generally the way you organise your data, i.e. your directory structure can help navigating the data and provide extra information, avoid confusion and the user ending up accessing the wrong data. As for many other cases the best file organisation will depend on the specific research project and the actual server where the data is stored. The global climate modelling intercomparison project has adopted a Data Reference Syntax (DRS), based on the controlled vocabularies (CVs) used in model metadata, to define their file names and directory structures. 
+Here we list a few guidelines and tips to help you decide.  
 
 ## General considerations
  Familiarise yourself with the storage system, make sure you are storing the files in the most appropriate place, get to know if the storage is backed up or not, what is your allocation, and also what rules or best practices apply.
```

Comment:
This could be a dot-point list, making it easier to read.

Reply:
Which? I'm guessing you're rereferring to "general considerations", not the new text? 
I can do that...

Reply:
Done in following commit.

Reply:
I'm a bit hazy, but I'm wondering if the storage part should be covered also in a separate section. We got a lot on the CMS wiki about storage, but as it was NCI related I didn't add it here. Another part which I didn't add although is more generic is the backup part.  Especially for the management guidelines we should cover backup, I can do that later this week :-)